BUILDING.rst

```diff
@@ -47,24 +47,24 @@ x86 platforms.
 On macOS, you may find it easiest to install Homebrew and install
 the following::
 
-    brew install autoconf automake bdw-gc --universal
+    brew install autoconf automake bdw-gc
+
+.. note:: Open Dylan doesn't yet support native code on Apple silicon and
+          therefore ``bdw-gc`` should be installed for the ``x86_64``
+          architecture. One way to accomplish this is to install an ``x86_64``
+          version of Homebrew with ``arch -x86_64 /bin/bash -c "$(curl -fsSL
+          https://raw.githubusercontent.com/Homebrew/install/HEAD/install.sh)"``
+          and install with that.
```

Comment:
I don't think this is a good recommendation. You're suggesting something that would alter their entire install of homebrew, right?

Reply:
ok, I dropped the brew-related changes and the sudo change but kept the changes to make version numbers less specific (so we don't have to keep updating them) and to point to release-with-batters.sh instead.

I guess the right way to install the GC is just to build it locally, a la release-with-batteries.sh?